segmentation/utils/data_utils.py

```diff
@@ -267,6 +276,34 @@ def tiffs_to_xr_labels(tiff_dir, output_dir=None, delimiter='_'):
     return im_xr
 
 
+# TODO: Add metadata for channel name (eliminates need for fixed-order channels)
+def generate_deepcell_input(data_xr, data_dir, nuc_channels, mem_channels):
+    """Saves nuclear and membrane channels into deepcell input format.
+
+    Inputs:
+        data_xr: xarray containing nuclear and membrane channels over many fov's
+        data_dir: location to save deepcell input tifs
+        nuc_channels: nuclear channels to be summed over
+        mem_channels: membrane channels to be summed over
+    Outputs:
+        Saves summed channels as multitiffs
+
+    """
+    for fov in data_xr.fovs.values:
+        out = np.zeros((data_xr.shape[1], data_xr.shape[2], 2), dtype="uint8")
+
+        # sum over channels and add to output
+        if nuc_channels:
+            out[:, :, 0] = np.sum(data_xr.loc[fov, :, :, nuc_channels].values.astype("uint8"),
```

Comment:
Same here. For microscopy data from other image platforms the dtypes can be much higher